packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/Errors/error-message/error-message.tsx

```diff
@@ -0,0 +1,28 @@
+import { noop as css } from '../../../helpers/noop-template'
+
+export type ErrorMessageType = string | React.ReactNode
```

Comment:
```suggestion
export type ErrorMessageType = React.ReactNode
```

`string` is already a member of `ReactNode`